### PR TITLE
Add lm_eval test for HPU

### PR DESCRIPTION
**Pull request by kzawora-intel**
Currently only testing Llama3.1-8b-Instruct with lm_eval on GSM-8K and IFeval. Both tests pass, can be expanded if needed.

### CODE REVIEW

File: tests/hpu/test_hpu_lmeval.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            "ifeval": {
+                'score': 0.804
+            }
+        }
+    }
+
+
+@pytest.mark.parametrize("task_cfg",
+                         [TaskConfigs.gsm8k_llama_cot, TaskConfigs.ifeval],
+                         ids=['gsm8k_llama_cot', 'ifeval'],
```
Comment (madamczykhabana) [inline tests/hpu/test_hpu_lmeval.py:115]: I think you can pass a function as 'ids'. Something like:
```python
def get_task_name(task_cfg):
    return task_cfg["task_name"]

...

@pytest.mark.parametrize("task_cfg",
                         [TaskConfigs.gsm8k_llama_cot, TaskConfigs.ifeval],
                         ids=get_task_name,
```

File: tests/hpu/test_hpu_lmeval.py
Comment (madamczykhabana) [inline tests/hpu/test_hpu_lmeval.py:1]: Perhaps we should move it to the extension repo?

File: tests/hpu/test_hpu_lmeval.py (hunk 1 above)
Comment (kzawora-intel) [reply tests/hpu/test_hpu_lmeval.py]: fixed, thanks!

File: tests/hpu/test_hpu_lmeval.py
Comment (kzawora-intel) [reply tests/hpu/test_hpu_lmeval.py:1]: Yeah, the PR got a little bloated now. I'll close it.

**Pull request closed without merging**